packages/gatsby-remark-embed-snippet/README.md

```diff
@@ -1,40 +1,115 @@
 # gatsby-remark-embed-snippet
 
-Embeds the contents of specified files within Prism-formatted HTML blocks.
+Embeds the contents of specified files as code-snippets.
 
-## Overview
+## Installation
+
+**Note**: This plugin depends on [gatsby-remark-prismjs](https://www.gatsbyjs.org/packages/gatsby-remark-prismjs/) and [gatsby-transformer-remark](https://www.gatsbyjs.org/packages/gatsby-transformer-remark/) plugins
+
+```shell
+# npm
+npm install --save gatsby-remark-embed-snippet gatsby-remark-prismjs gatsby-transformer-remark
+
+# yarn
```

Comment:
We recommend npm instructions throughout the docs, so we should remove these yarn instructions for consistency. Here's a similar README: https://www.gatsbyjs.org/packages/gatsby-image/#install

Reply:
i removed yarn instructions